src/modules/BootSupportCheck.rb

```diff
@@ -86,6 +86,11 @@ def CorrectLoaderType
           return true if lt == "grub2"
         end
       end
+
+      if Arch.aarch64
+        return true if lt == "grub2-efi"
+      end
```

Comment:
again above is problem in `return true if lt == "grub2"` as it is not correct for aarch64 for which it should not be allowed combination.